internal/audit/hub/conf.go

```diff
@@ -42,6 +47,8 @@ type Conf struct {
 type IngestConf struct {
 	// MaxBatchSize defines the max number of log entries to send in each Ingest request.
 	MaxBatchSize uint `yaml:"maxBatchSize" conf:",example=32"`
```

Comment:
Shall we hide this as well because it doesn't make much sense for the users to set it? In fact, I think we should ignore it completely in the code because trying to satisfy both the `batchSize` and 'batchSizeBytes` constraints makes it difficult to do things efficiently. In the case of a busy PDP, trying to stick to the `batchSize` limit means we'd be making a lot more API calls than necessary.

Reply:
Might be worth still enforcing a hard limit in the code, at the very least, so we can [reuse the same memory for batching](https://github.com/cerbos/cerbos/blob/main/internal/audit/hub/hub.go#L236-L241). IIRC, we implemented this because it quite significantly reduced allocations and GC overhead 🤔

Reply:
Actually, I forgot there's a [limit on the number of entries](https://github.com/cerbos/cloud-api/blob/706862567a9d26847c736aa7e4e24e2e768a9661/protos/cerbos/cloud/logs/v1/logs.proto#L41-L44) so we'd still need to ensure that the outgoing request doesn't exceed that. Let's just treat that value as a constant and not a user-configurable value.